.github/workflows/ci.yml

```diff
@@ -17,14 +17,7 @@ jobs:
       matrix:
         java-version:
           - 1.8
-          - 9
-          - 10
```

Comment:
Is there any concern for the fact that 10 is broken?

Reply:
I'd say probably not, it's pretty old and not a LTS release. It might be worth also tracking the newest release in addition as well in the future.

For right now, restricting to LTS releases should unblock CI from failing.

Reply:
OK sounds good.